### DISCUSSION TITLE
Welcome to visit the "Artificial Intelligence for Beginners" course taught by AI

### DISCUSSION THREAD

**Discussion by sailxjx** [microsoft/AI-For-Beginners#347] [Show and tell]
I've created a series of AI courses that allow for conversation with AI, using the content from this repo.

When you learn this course from the platform, the AI teacher will guide you through the lessons step by step, and you can ask anything about the topics covered.

The course is free, everyone is welcome to visit.

[Artificial Intelligence for Beginners](https://codekidz.ai/lesson-intro/artificial-i-36275c)

And also for someone is interested in Generative AI

[Generative AI for Beginners](https://codekidz.ai/lesson-intro/generative-a-362093)

**Comment by ellielo** [comment]
New to AI? Stick with regular practice along with real project work. Begin by exploring basics - think supervised compared to unsupervised methods - followed by trying out tiny tests of your own. Notes and written guides made using AI tools might assist, though clear phrasing stays essential. Pass early versions through [clever ai humanizer](https://cleverhumanizer.ai/) so complex ideas feel simpler to grasp while avoiding that robotic tone. Faster than a coffee break, it fits right into your study rhythm. What you say matters just as much as what you know.
